frontend/src/selectors/staticContentSelectors.js

```diff
@@ -70,3 +70,8 @@ export const getCommodityOptionHash = createSelector(
       {}
     )
 );
+
+export const getDropdownCommodityOptions = createSelector(
```

Comment:
ah, we will have to create a test for this. I forgot about this selector, I can help with that test.